API/endpoints/CloudRecording.js

```diff
@@ -75,9 +73,8 @@ CloudRecording.listMeetingRecordings.scopes = [
  *   returned from a single API call
  * @param {string} [options.nextPageToken] - token used to pageinate
  *   through large result sets
- * @param {string} [options.searchTrashFor] - Indicate type of
- *   cloud recording to retreive from the trash.
- *   options - {'meeting_recordings', 'recording_file'}
+ * @param {string} [options.searchTrash=false] - set to true to retrieve
+ *   meeting recordings from the trash.
  * @param {string|Date} [options.startDate=6 months before today]
  *   - string accepted by JS Date constructor or instance of Date object.
  *   Date needs to be within past 6 months. Time data is discarded
```

Comment:
Maybe "Time data (hours and seconds) will be discarded" or "Date will be truncated to year, month, and day" to be unambiguous

Reply:
My vote is for the first one.